### PR TITLE
address PR #31 feedback

### PR DESCRIPTION
**Pull request by strcrzy**
- dynamically restore sign up button text to original value
- break callback handler into two functions

### CODE REVIEW

File: stream/src/js/stream.js [hunk 1]
```diff
@@ -6,24 +6,26 @@ $(document).ready(function() {
...
-                // Keep the form as is for resubmission
-                form.find('input.btn').addClass('btn-warning').val('Error. Try again?');
-                setTimeout(function() {
-                    form.find('input.btn').removeClass('btn-warning').addClass('btn-primary').val('SIGN UP');
-                }, 5000);
-            }
+        var handle_success = function(form) {
+            // Let user know that it's done
+            var button = form.find('input.btn')
+            var sign_up_text = button.val()
```
Comment (ryanhobsonsmith) [inline stream/src/js/stream.js]: Believe you can move the button and sign_up_text variable declarations up to where you define the form var (would reduce code duplication and protect against possible race-condition when reading/setting button text).

Comment (strcrzy) [reply stream/src/js/stream.js]: 👍 thanks

**Comment by ryanhobsonsmith** [comment]
Looks good 👍 , Assume you've tested that API calls actually work and go through? I didn't actually run the code as part of my review.

**Comment by strcrzy** [comment]
i have not tested the API calls, it's unclear if they have ever been tested but they should, especially as they were reverse engineered /cc @kpenning

**Comment by liyanchang** [comment]
Alex and I have been testing this by entering in data and then getting an email confirmation. Same confirmation if you sign up at https://go.peoplepower.org/

**Pull request merged**